cmd/performance-profile-creator/cmd/root.go

```diff
@@ -183,14 +190,95 @@ func getClusterData(mustGatherDirPath string) (ClusterData, error) {
 	return cluster, nil
 }
 
-func showClusterData(cluster ClusterData) {
+// NUMACellInfo describe a NUMA cell on a node
+type NUMACellInfo struct {
+	ID       int   `json:"id"`
+	CoreList []int `json:"cores"`
+}
+
+// NodeInfo describe a Node in a MCP
+type NodeInfo struct {
+	Name           string         `json:"name"`
+	HTEnabled      bool           `json:"smt_enabled"`
+	NUMACellsCount int            `json:"numa_cells_count"`
```

Comment:
Is this really needed as we can compute this with by len(NUMACells) and don't see it being used for logging?

Reply:
nope, not needed. Will remove.